scheduler/test/testdata.go

```diff
@@ -143,6 +146,54 @@ var PodForHandleTest = &v1.Pod{
 	Spec: v1.PodSpec{NodeName: NodeSmall.Name},
 }
 
+var PodForHandlePod = &v1.Pod{
+	ObjectMeta: apimeta.ObjectMeta{
+		Name:      "handle-pod",
+		Namespace: "test",
+		UID:       "384900cd-dc7b-41ec-837e-9c4c1762363e",
+	},
+	Spec: v1.PodSpec{NodeName: NodeSmall.Name},
+}
+
+// waitingPod implements the framework.WaitingPod interface
+type waitingPod struct {
+	pod            *v1.Pod
+	pendingPlugins map[string]*time.Timer
+	s              chan *framework.Status
+	mu             sync.RWMutex
+}
+
+func (wp *waitingPod) GetPod() *v1.Pod {
+	return wp.pod
+}
+
+func (wp *waitingPod) GetPendingPlugins() []string {
+	wp.mu.RLock()
+	defer wp.mu.RUnlock()
+	var plugins []string
+	for plugin := range wp.pendingPlugins {
+		plugins = append(plugins, plugin)
+	}
+	return plugins
+}
+
+func (wp *waitingPod) Allow(pluginName string) {
+	wp.mu.Lock()
+	defer wp.mu.Unlock()
+}
+
+func (wp *waitingPod) Reject(reason string, msg string) {
+	wp.mu.Lock()
+	defer wp.mu.Unlock()
+}
+
+// Create the test WaitingPod
+var WaitingPod = &waitingPod{
+	pod:            nil,
+	pendingPlugins: make(map[string]*time.Timer),
+	s:              make(chan *framework.Status, 1),
+}
```

Comment:
do we use them?

Reply:
`waitingPod` is called in https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/blob/fddccd7185ba0a7cf30095aa0f3cbbd94a10d820/scheduler/test/util.go#L82 but no need WaitingPod